.github/ISSUE_TEMPLATE/add_plugin.md

```diff
@@ -0,0 +1,54 @@
+---
+name: Add plugin
+about: Request to add your plugin to the Marketplace
+title: "Add $REPOSITORY_NAME to Marketplace"
+labels: Plugin/New
+assignees: hanzei
+
+---
+<!--
+Thank you very for submitting your Plugin! It will go through a review process to make sure it follow the quality standard  of the Marketplace. This process might take a couple of weeks bepending on how many changes are needed.
+Read https://developers.mattermost.com/extend/plugins/community-plugin-marketplace/ before submitting your plugin.
```

Comment:
Doesn't resolve yet, but I'm assuming that's because it's just under review.

Reply:
Once https://github.com/mattermost/mattermost-developer-documentation/pull/524 is merged, it should resolve.